app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set correctly in your environment. If it's missing, the script will fail to authenticate with the Weather API.